CMSgov/design-system
# [WNMGDS-821] Move lists to typography page

## Summary
Moved list html examples and guidance from it's own component page to the typography page

### Added
- Added list information to the typography page

### Removed
- Removed the list page from the components section 


## How to test
Check out the demo site and see the following
- Lists is no longer listed as a component
- List info can be found under Styles > Typography
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-821
